Automattic/jetpack
# Protect: Add unignore threat capabilities

## Description
Allows users to unignore threats found in the ignored section of their scan history.

Reliant on:
- Automattic/vaultpress#106996
- D153672

## Proposed changes:

* Adds an endpoint responsible for calling the wpcom endpoint to trigger the unignore change
* Adds a conditional unignored button to ignored threats in scan history UI

### Other information:

## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1250

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch and start up Jurassic Tube
* Install Protect add fixable and non-fixable threats and run a scan to detect them
* Ignore several threats and proceed to History and filter by `Ignored`
* Select `Unignore threat` and verify that the threat is added back to the current threats list
* Check `vp-global` to ensure the `vp_scan_results` entry was updated as expected
* Ensure no regressions are introduced
